packages/components/pagination/src/components/sizes.vue

```diff
@@ -7,6 +7,7 @@
       :size="size"
       :teleported="teleported"
       :validate-event="false"
+      :append-to="appendDropdownTo"
```

Comment:
This seems to be a duplicate of `teleported`.

Reply:
`Teleported` is a boolean for appending element to `body`, `append-to` allows to append anywhere, not just `body`

Reply:
@btea if that's okay, could you approve this PR?

Reply:
I can change it to `appendSizeTo`